src/util/system.cpp

```diff
@@ -79,6 +79,7 @@
 #include <fstream>
 #include <map>
 #include <memory>
+#include <optional>
```

Comment:
22766; don't understand why it's partial. Missing changes, appear to be able to be done

Reply:
Will correct excluded commit in description, b8c069b7a952e326d2d974cc671889d1a3b38aa4 is included, c5d7e34bd9a4ad752c5ec88032420e2e90ab17ab isn't. The latter has been excluded as flags like `-legacy` and anything `-signet*`, which are affected by the scripted-diff, haven't been implemented yet.